pilot/lib/pilot/build_manager.rb

```diff
@@ -96,6 +96,15 @@ def distribute(options, build: nil)
       build.auto_notify_enabled = config[:notify_external_testers]
 
       return if config[:skip_submission]
+      if options[:reject_previously_submitted_build]
+        waiting_for_review_build = Spaceship::TestFlight::Build.all_waiting_for_review(app_id: build.app_id, platform: :ios).first
```

Comment:
Do we want to throw an error here if there is more than one? 🤔 Calling `.first` seems save enough but would be confusing **if** there was two and then it failed for this reason 😬

Reply:
@joshdholtz I don't think there could be more than one 'waiting for review'. Apple doesn't allow more than one build to be submitted for review. So this array would always have max 1 value.